clerk/users.go

```diff
@@ -108,14 +108,18 @@ func (s *UsersService) Create(params CreateUserParams) (*User, error) {
 }
 
 type ListAllUsersParams struct {
-	Limit             *int
-	Offset            *int
-	EmailAddresses    []string
-	ExternalIDs       []string
-	PhoneNumbers      []string
-	Web3Wallets       []string
-	Usernames         []string
-	UserIDs           []string
+	Limit          *int
+	Offset         *int
+	EmailAddresses []string
+	ExternalIDs    []string
+	PhoneNumbers   []string
+	Web3Wallets    []string
+	Usernames      []string
+	UserIDs        []string
+	// OrganizationIDs filters users that have memberships to the given organizations. For each organization ID, the
```

Comment:
Copy of the docs from https://clerk.com/docs/references/backend/user/get-user-list

Reply:
💯 I love seeing the docs here in the SDK. As a developer, if I can get the answer to "what is this field" without having to leave the IDE, i'm much more efficient.

It breaks our existing pattern, so others can weigh in on whether we should start doing it now in this PR, wait until we have some programmatic solution to add comments in bulk and keep them up-to-date, or object to the premise 😄 . But I'd be OK with including it here.